starwars/src/components/Card/CardContainer.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import "../StarWars.css";
+import CharacterStats from "./CharacterStats";
+
+const CardContainer = props => {
+  console.log(props.character);
+  return props.character.map(item => {
+    return (
+      <div className="character-card">
```

Comment:
Would put this and the line below into CharacterStats component to look cleaner and makes it easier to find when doing styling.